.github/workflows/ci-check.yml

```
@@ -0,0 +1,37 @@
+name: local pre-commit
```

```suggestion
name: Test if pre-commit prettier works
```
To really test if prettier works it would be nice to have some example files so it not only tries to format the yaml.
Would be nice if it formats `xml` with the `@prettier/plugin-xml` plugin.
The real different of this fork is that it is able to load prettier plugins "easier".